File: week-04/day-01/sharpie.py
class Sharpie(object):
    def __init__(self, color, width, ink_amount=100):
        self.color = color
        self.width = width
        self.ink_amount = ink_amount


    def use(self):
        self.ink_amount -= 1.5


dog  = Sharpie("yellow", 10)
dog.use()
print(dog.ink_amount)

File: week-08/day-01/sharpie.js
function Sharpie() {
    this.color = 'red',
    this.width = 10,
    this.inkAmount = 100;
    this.useSharpie = function()  {
        this.inkAmount -= this.width;
    }
}
let reddy = new Sharpie();
while (reddy.inkAmount > 0) {
    reddy.useSharpie();
    console.log(reddy.inkAmount)
}
